src/components/loading-indicator.js

```diff
@@ -17,6 +18,12 @@ const dimensions = {
 };
 
 const LoadingIndicator = ({ centered, size, type }) => {
+  const ready = useTimeout(300);
```

Comment:
Also slipped in a UX improvement. This helps to prevent the quick blinking of loading indicators, seen when response times are very fast (e.g. via cached response).